apps/dashboard/src/components/workflow-editor/steps/edit-step-sidebar.tsx

```diff
@@ -1,28 +1,61 @@
-import { useNavigate } from 'react-router-dom';
-import { RiEdit2Line } from 'react-icons/ri';
+import { useLayoutEffect, useMemo } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+// eslint-disable-next-line
+// @ts-ignore
```

Comment:
I don't think this is needed. I haven't noticed any errors when using it.

Reply:
will take a look